src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -65,9 +56,9 @@ describe('generateConfluenceMarkdown function', () => {
 
         const twoIssues = mockDependencyMap.set(dependency, fakeIssues);
         const results = mariner.generateConfluenceMarkdown(twoIssues);
-        expect(results).toContain(`h3. ${dependency}`);
-        expect(results).toContain(`|[${fakeIssues[0].title}|${fakeIssues[0].url}]|`);
-        expect(results).toContain(`|[${fakeIssues[1].title}|${fakeIssues[1].url}]|`);
+
```

Comment:
Shouldn't we still expect the dependency header to be included?

Reply:
just added header to tests